Author: Vincent Prat <vinceprat@free.fr>
Description: Use ConfigParser module instead of the one embedded in distutils.config
--- a/setup.py
+++ b/setup.py
@@ -21,8 +21,11 @@
 from astropy_helpers.version_helpers import generate_version_py
 
 # Get some values from the setup.cfg
-from distutils import config
-conf = config.ConfigParser()
+try:
+    from configparser import ConfigParser
+except:
+    from ConfigParser import ConfigParser
+conf = ConfigParser()
 conf.read(['setup.cfg'])
 metadata = dict(conf.items('metadata'))
 
